queries/julia/textobjects.scm

```diff
@@ -1,16 +1,24 @@
-(compound_expression) @block.outer    ;; begin blocks
+(compound_expression) @block.outer    ; begin blocks
 (let_statement)       @block.outer
 
-(call_expression) @call.outer
+(call_expression
+  (identifier)
+  (argument_list) @parameter.outer) @call.outer
+
+(struct_definition) @class.outer    ; not classes, but close enough
 
 (comment) @comment.outer
 
 (if_statement) @conditional.outer
 
-(function_definition) @function.outer
-(function_expression) @function.outer    ;; lambdas
+(function_definition
+  name: (identifier)
```

Comment:
why do we need the `name` in the query? Can't the function as well be anonymous and if not then `name: (identifier)` does not really add a benefit.-

Reply:
Ok. It's a function_expression without the name.

Reply:
I actually forgot to check that case 😬

Good thing it's covered by the other query, although it does seem a bit confusing. Not sure why the grammar would be defined that way.